runners/jstorm/src/main/java/org/apache/beam/runners/jstorm/translation/ParDoBoundTranslator.java

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
<!--thread_id:cc_130004480_t; commit:6d1d2030507abb560abfea721679f2f92734618c; resolved:0-->
<!--section:context-quote-->
> **peihe** wrote:
> I think we can remove this class because SingleOutput will be translated to MultiOutput by default.

<!--section:body-->
This is true. Only multi output ParDo is a real Beam primitive. You can optionally use a replacement to have a specialized single output ParDo if that gains performance or makes some UI easier to read.